index.html

```diff
@@ -24,7 +15,7 @@ <h5 class="header col s12 light">Welcome to the Trasies</h5>
         <div class="row center">
             <a href="http://materializecss.com/getting-started.html" id="create"
                class="btn-large waves-effect waves-light orange">Create</a>
-            <a href="join.html" id="join" class="btn-large waves-effect waves-light blue">Join</a>
+            <a href="/join" id="join" class="btn-large waves-effect waves-light blue">Join</a>
```

Comment:
So using /join will just make it so you navigate to _website_(href here)?

Reply:
/join is the path we're hosing on the go server, so this link will point to that page.